ddp-workspace/projects/ddp-dsm-ui/src/app/interceptors/Http-interceptor.service.ts

```diff
@@ -23,13 +23,16 @@ export class HttpInterceptorService implements HttpInterceptor {
               private readonly authService: Auth) {}
 
   intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
-    return next.handle(req).pipe(
+    var ignoreError = req?.headers?.has(HttpInterceptorService.InterceptorSkipHeader);
+    // InterceptorSkipHeader is only used internally to DSM.  don't pass it on to servers.
+    const cleanedHeaders = req.clone().headers.delete(HttpInterceptorService.InterceptorSkipHeader);
+    var cleanReq = req.clone({ headers: cleanedHeaders });
```

Comment:
Seems like a lot of angular code out there does this, although mostly it's do add auth headers.  In this case, we clone the headers, remove the interceptor skip header, and then run the request.